dataactbroker/handlers/fileHandler.py

```diff
@@ -290,7 +303,8 @@ def getStatus(self):
                     else:
                         jobInfo["duplicated_headers"] = []
                     jobInfo["error_type"] = self.interfaces.errorDb.getErrorType(jobId)
-                    jobInfo["error_data"] = self.interfaces.errorDb.getErrorMetricsByJobId(jobId,self.jobManager.getJobType(jobId)=='validation',self.interfaces)
+                    jobInfo["error_data"] = self.interfaces.errorDb.getErrorMetricsByJobId(jobId,self.jobManager.getJobType(jobId)=='validation',self.interfaces, severityId=self.interfaces.validationDb.getRuleSeverityId("fatal"))
```

Comment:
Since this is in a `for job in jobs` loop, might be another opportunity to save the severity ids to variables before the loop to reduce the number of db calls (and also the `getJobType(jobId)=="validation"`).